l10n_br_account_product/sped/nfe/document.py

```diff
@@ -251,7 +251,12 @@ def _emmiter(self, invoice, company):
             company.zip or '')
         self.nfe.infNFe.emit.enderEmit.cPais.valor = (
             company.country_id.bc_code[1:])
-        self.nfe.infNFe.emit.enderEmit.xPais.valor = company.country_id.name
```

Comment:
Renato put this if to make life easier for those who are testing.

At least here at Kmee, we use the base very much in English, without installing the translation. Leaves the process much faster...

And it can also happen that an unsuspecting person tries to use the system in homologation without having installed the translation.